workers/unity/Packages/com.improbable.gdk.core/Logging/LogEvent.cs

```diff
@@ -20,14 +26,28 @@ public LogEvent WithField(string key, object value)
             return this;
         }
 
+        public LogEvent WithContext(UnityEngine.Object context)
```

Comment:
These would benefit greatly from XMLDoc comments to help people use the public API :)

Reply:
Added XMLDoc